overviewer_core/src/mc_id.h

```diff
@@ -377,7 +377,10 @@ enum mc_block_id {
     block_honey_block = 11504,
     block_sweet_berry_bush = 11505,
     block_campfire = 11506,
-    block_bell = 11507
+    block_bell = 11507,
+    // 1.16 blocks below
+    block_crimson_slab = 11508,
+    block_warped_slab = 11509
```

Comment:
Can you use lower IDs? We currently store block sprites in an array, so if we keep making them higher we have a lot of unused space in it. Thanks!

Reply:
Hello, I just increased the IDs as it was done before, do you mean filling ids from 1806 to 11336?

Reply:
Use 1000+, I added some 1.16 blocks there before. Don't use IDs between 1792 and 2047, those are all considered walls due to the wall checking function using a bit operation.